README.md

```diff
@@ -25,9 +21,6 @@ These options allow you to specify an AWS S3 bucket path where the browse image
 2. Installing `gdal` via pip requires that `git`, `gcc`, and `g++` are installed on your system.
```

Comment:
Bit of a nit-pick, but this issue isn't something a user needs to worry about if they're installing via Conda, which is what we recommend. Feel free to change if you agree.